packages/browser-wallet/src/popup/pages/Web3ProofRequest/Display/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { CredentialSchemaSubject } from '@shared/storage/types';
+import { ClassName } from 'wallet-common-helpers';
+
+export function getPropertyTitle(attributeTag: string, schemaSubject: CredentialSchemaSubject) {
+    // TODO use localization here
```

Comment:
I guess we should add this to the slack list? :smile:

Reply:
Yes please do :upside_down_face: